c/pthread-ext/06_ticket_true-unreach-call.c

```diff
@@ -1,5 +1,8 @@
 extern void __VERIFIER_assume(int);
 extern void __VERIFIER_error() __attribute__ ((__noreturn__));
+int __global_lock;
+void __VERIFIER_atomic_begin() { __VERIFIER_assume(__global_lock==0); __global_lock=1; return; }
```

Comment:
So I just copied this pattern from the `pthread-wmm` benchmarks (for example, see https://github.com/sosy-lab/sv-benchmarks/blob/master/c/pthread-wmm/mix000_power.oepc_false-unreach-call.c).

`pthread-wmm` is the only Concurrency sub-group that actually does use the newer, non-depreciated `__VERIFIER_atomic_begin/end`, so I figured I would just implement these new ones in the same way that `pthread-wmm` had done.

How should we proceed on this?